doc/sphinx-guides/source/container/app-image.rst

```diff
@@ -0,0 +1,220 @@
+
+- | ... a different tag only: add ``-Dapp.image.tag=tag``.
+  | *Note:* default is ``unstable``
+- | ... a different image name and tag: add ``-Dapp.image=name:tag``.
+  | *Note:* default is ``gdcc/dataverse:${app.image.tag}``
+- ... a different image registry than Docker Hub: add ``-Ddocker.registry=registry.example.org`` (see also
+  `DMP docs on registries <https://dmp.fabric8.io/#registry>`__)
+- | ... a different base image tag: add ``-Dbase.image.tag=tag``
+  | *Note:* default is ``unstable``
+- | ... a different base image: add ``-Dbase.image=name:tag``
+  | *Note:* default is ``gdcc/base:${base.image.tag}``. See also :doc:`base-image` for more details on it.
+
+Automated Builds & Publishing
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+See note above at "Supported Image Tags".
+
+.. _app-multiarch:
+
+Processor Architecture and Multiarch
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This image is created as a "multi-arch image", supporting the most common architectures Dataverse usually runs on:
+AMD64 (Windows/Linux/...) and ARM64 (Apple M1/M2), by using `Maven Docker Plugin's BuildX mode <https://dmp.fabric8.io/#build-buildx>`_.
+
+Building the image via ``mvn -Pct package`` or ``mvn -Pct install`` as above will only build for the architecture of
+the Docker machine's CPU.
+
+Only ``mvn -Pct clean deploy -Ddocker.platforms=linux/amd64,linux/arm64`` will trigger building on all enabled architectures.
+Yet, to enable building with non-native code on your build machine, you will need to setup a cross-platform builder.
+
+On Linux, you should install `qemu-user-static <https://github.com/multiarch/qemu-user-static>`__ (preferably via
+your package management) on the host and run ``docker run --rm --privileged multiarch/qemu-user-static --reset -p yes``
+to enable that builder. The Docker plugin will setup everything else for you.
+
+
+
+.. _app-tunables:
+
+Tunables
+++++++++
+
+The :doc:`base-image` provides a long list of possible options to tune many aspects of the application server, and,
+as the application image builds upon it, :ref:`Base Image Tunables <base-tunables>` apply to it as well.
+
+In addition, the application image provides the following tunables:
+
+.. list-table::
+    :align: left
+    :width: 100
+    :widths: 10 10 10 50
+    :header-rows: 1
+
+    * - Env. variable
+      - Default
+      - Type
+      - Description
+    * - ``MP_CONFIG_PROFILE``
+      - ``ct``
+      - String
+      - Set to switch the activated *MicroProfile Config Profile*. Note that certain defaults will not apply any longer.
+        See :ref:`:ApplicationServerSettings` for details.
+    * - ``dataverse_*`` and ``doi_*``
+      - \-
+      - String
+      - Configure any :ref:`jvm-options` not yet *MicroProfile Config* enabled with this magic trick.
+
+        1. Simply pick a JVM option from the list and replace any ``.`` with ``_``.
+        2. Replace any ``-`` in the option name with ``__``.
+    * - ``DATAVERSE_MAIL_HOST``
```

Comment:
Why is the SMTP configuration different from the rest?

Reply:
Because this is a hack. It's ugly. It's bad. But until we don't resolve #7424, we will have to stick with the hack.

To give some technical background: env vars are case sensitive. So these vars will not be picked up by the hack to inject system properties into the domain configuration, where they would have no effect. Already writing them in uppercase is kind of related to what is happening for #7424 - the names are not random, but equivalent to what will be added as `JvmSetting`. Which in turn means anyone using this will have a non-breaking change, it's directly compatible.

Reply:
Simply because those are already MPCONFIG enabled. They are not specific to the image.

Reply:
Is there anything to fix here? My take is that we can revisit SMTP after we address the following issue Oliver pointed out above:

- #7424

Reply:
That would be a deletion, BTW. Get rid of the hacky stuff.